third-party/xen/Makefile

```diff
@@ -0,0 +1,16 @@
+
+PKG_NAME := xen
+PKG_VER  := stable-4.13
+
+PKG_SOURCES := git://xenbits.xen.org/xen.git
```

Comment:
Can we use source archive bundle, i.e. from https://xenproject.org/downloads/xen-project-archives/xen-project-4-13-series/xen-project-4-13-0/ (https://downloads.xenproject.org/release/xen/4.13.0/xen-4.13.0.tar.gz)

There is a problem with git sources, it requires a whole mini-language to specify how to clone. For example,  revision/branch `aaa` of project A and revision `bbb` of project B. And for one one of them you want `--depth 1`. I don't know even how could it be specified in the Makefile prettily.

So we tend to use source archive bundles if possible. It looks like it's possible here too.

Reply:
They don't share md5 sum. Is this correct to use self-computed sum?

Reply:
Yes, of course, this is perfectly fine

Reply:
Done in #1721.